### PR TITLE
feat: print the timeout from the CR for the workspace idle/runtimeout…

### PR DESCRIPTION
**Pull request by dkwon17**
… warning page

Signed-off-by: dkwon17 <dakwon@redhat.com>

### What does this PR do?
Reads the `spec.devEnvironments.secondsOfInactivityBeforeIdling` and `spec.devEnvironments.secondsOfRunBeforeIdling` for the idle timeout and run timeout warning pages respectively.

<details>
<summary>Workspace inactivity</summary>

Screen Shot 2022-08-05 at 11 27 15 AM

</details>

<details>
<summary>Workspace runtime limit </summary>

Screen Shot 2022-08-05 at 11 33 55 AM

</details>

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/21578

#### Testing the 'Workspace inactivity' error page
1. Set the workspace inactivity timeout to a smaller value by setting checluster.spec.devEnvironments.secondsOfInactivityBeforeIdling to, for example, 120.

2. Start a workspace. Once the editor opens, let it timeout by not interacting with it for about 120 seconds.

3. Once the workspace times out, refresh the page

4. The error page should mention that the idle timeout is 2 minutes.

#### Testing the 'Workspace run timeout' error page
1. Set the workspace run timeout to a value smaller than the inactivity timeout by setting `checluster.spec.devEnvironments.secondsOfRunBeforeIdling` to, for example, `80`.

2. Start a workspace. After the workspace starts, the workspace should terminate in about 80 seconds.

3. Once the workspace times out, refresh the editor page

4. The error page should mention that the run timeout is 1 minute and 20 seconds.

#### Release Notes

The workspace inactivity and workspace run timeout warning pages now display their respective timeout durations.

### CODE REVIEW

File: packages/dashboard-frontend/src/Layout/ErrorReporter/Issue/index.tsx [hunk 1]
```diff
@@ -13,8 +13,9 @@
 import { TextContent, Text, TextVariants } from '@patternfly/react-core';
 import { InfoIcon, WarningTriangleIcon } from '@patternfly/react-icons';
 import React from 'react';
+import { workspace } from 'vscode';
```
Comment (akurinnoy) [inline packages/dashboard-frontend/src/Layout/ErrorReporter/Issue/index.tsx:16]: Unused import:

```suggestion
```

**Review by akurinnoy — APPROVED** [review]
LGTM

**Review by olexii4 — APPROVED** [review]

**Comment by dkwon17** [comment]
Thank you for the reviews, I've removed the unused import

**Review by ibuziuk — APPROVED** [review]

**Comment by dkwon17** [comment]
Planning to squash and merge

**Pull request merged**